Automattic/wp-calypso
# Signup: Remove stroke value on WP logo

#### Proposed Changes

* Revert #64517
* Drop stroke css instead 

#### Testing Instructions

* http://calypso.localhost:3000/start/domains
* Observe the logo change

Before 
![Screenshot 2022-08-05 at 15-13-03 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/183006264-e827ceb3-2b60-458b-a3b7-b544a77c448e.png)

After
![Screenshot 2022-08-05 at 15-12-42 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/183006274-7dcb3aec-8995-4cd7-bba4-6cb58750f4f2.png)

Before 
![Screenshot 2022-08-05 at 15-20-40 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/183006970-65aedffe-ee8b-4089-a440-f2f295f88930.png)

After
![Screenshot 2022-08-05 at 15-21-39 Create a site — WordPress com](https://user-images.githubusercontent.com/811776/183007075-151b3a17-d118-488b-ab00-59b7ea0c6a3b.png)
